Sign_in.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Sign in to the Saucedemo Application
+
+As a user, I want to sign in so I can use application functionality.
```

Comment:
No need to add it here (it will be recognized as an error during the automation process).
Just a comment, not a mistake:)